packages/mui-material/src/TextareaAutosize/TextareaAutosize.tsx

```diff
@@ -153,6 +154,21 @@ const TextareaAutosize = React.forwardRef(function TextareaAutosize(
     if (heightRef.current !== outerHeightStyle) {
       heightRef.current = outerHeightStyle;
       textarea.style.height = `${outerHeightStyle}px`;
+
+      // This is a workaround for Safari/WebKit not reflowing text when the textarea height changes
+      // Force Safari to reflow the text by  manipulating the textarea value
+      const containerDocument = ownerDocument(textarea);
+      const selectionStart = textarea.selectionStart;
+      const selectionEnd = textarea.selectionEnd;
+      const tempValue = textarea.value;
+      textarea.value = '';
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      textarea.offsetHeight;
```

Comment:
```suggestion
      void textarea.offsetHeight;
```
Does this work instead of eslint-disable?

Reply:
Yes, it worked without disabling the ESLint

Reply:
Yes, it worked without eslint-disable